internal/apiutil/errors.go

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package apiutil
+
+import (
+	"github.com/mainflux/mainflux/pkg/errors"
+)
```

Comment:
```go
import "github.com/mainflux/mainflux/pkg/errors"
```
Also, a nice comment in this file explaining what these errors represent would be good.